### ISSUE TITLE
Heater effect not fading

### ISSUE THREAD

**Issue by danicedenv** [julianschill/klipper-led_effect#133]
![image](https://user-images.githubusercontent.com/41544272/220728788-be1f5d95-e5a7-4356-b78e-f4463910d831.png)
Hey everyone so I'm having issues with the heater effect. I don't know if I'm doing something wrong but instead of fading it simply swaps to the second colour as soon as the temperature is reached. Thanks in advance.

**Comment by julianschill** [comment]
The last color is just used to switch to when the temperature is reached. So you have only defined one color to fade through. Add another color.

**Comment by danicedenv** [comment]
I get it thanks. I think what confused me is that on the doc it says "Once it has been met, the last color of the gradient..." maybe it would be better to say the last colour of the palette. Brilliant work so far, thanks for all this.

**Comment by julianschill** [comment]
You are right. I will either change the behaviour of the heater effect or improve the wording in the documentation.
